docs/source/tutorials.rst

```diff
@@ -445,18 +445,19 @@ More information about the COSMOS catalog
 
-Internally, BTK uses galsim to draw the galaxies. In particular, it instantiates a ``galsim.COSMOSCatalog``, that requires both catalogs. Yet, this object enables galsim to draw galaxies in two different modes that do not use the two catalogs in the same way: the parametric mode uses information of the second catalog while the 'real' mode uses information of the first catalog (and the actual galaxy and PSF stamps). You can refer to the galsim `documentation <https://galsim-developers.github.io/GalSim/_build/html/real_gal.html>`_ for more details. In BTK, we use only the 'real' drawing mode, so that the information of the second catalog is not necessary, even if the file must exist to instantiate the ``CosmosCatalog`` and ``galsim.COSMOSCatalog`` objects.
-However, BTK still retrieves the ``flux_radius`` information from this catalog, in order to compute an estimate of the size of each source and to measure deblending performance depending on the source sizes. Thus, the following conditions must be satisfied when providing custom COSMOS data to BTK:
+Internally, BTK uses galsim to draw the galaxies. In particular, it instantiates a ``galsim.COSMOSCatalog``, that requires both catalogs. Yet, this object enables galsim to draw galaxies in two different modes that do not use the two catalogs in the same way: the parametric mode uses information of the second catalog while the 'real' mode uses information of the first catalog (and the actual galaxy and PSF stamps). You can refer to the galsim `documentation <https://galsim-developers.github.io/GalSim/_build/html/real_gal.html>`_ for more details.
+When using 'real' galaxies, most of the information of the second catalog is not necessary, but the file must exist to instantiate the ``CosmosCatalog`` and ``galsim.COSMOSCatalog`` objects (and in practice, BTK uses the ``flux_radius`` column to compute an estimate of the size of each source used for performance evaluation measures, so the second catalog should contain at least this column).
+Custom COSMOS catalogs to draw 'real' galaxies should thus satisfy the following conditions:
 
 1. The second catalog should contain at least the ``flux_radius`` column,
 
 2. The first catalog should contain the same columns than the official COSMOS data release
 
 3. The galaxy and PSF stamps should be provided and accessible.
 
-4. (optional) One of the two catalogs can contain multiband magnitudes using the format just described.
+4. (optional) One of the two catalogs can contain multiband magnitudes using the format described above.
```

Comment:
Here I refer to the format described in [this section](https://github.com/LSSTDESC/BlendingToolKit/blob/6bc38c9403d615740e4434d81a2ed471c9798569/docs/source/tutorials.rst#using-different-magnitudes-for-each-band), i.e. "sn_fn" with sn and fn being the survey name and filter name, respectively. Is there a way to make a link to the section in the doc ? This may make things more clear.